source/data-model.txt

```diff
@@ -0,0 +1,118 @@
+.. _data-model:
+
+===================
+Define a Data Model
+===================
+
+.. contents:: On this page
+   :depth: 1
+
+.. toctree::
+
+   Schemas </schemas>
+   Relationships </schemas/relationships>
+   Values & Secrets </values-and-secrets>
+
+Overview
+--------
+
+Your App's data model is a description of the data that your App uses.
+Your data model helps to ensure that your data is well-formed, contains
```

Comment:
Nit: the data model doesn't actually ensure that the data is "well-formed." If the data model isn't well-formed, the data won't be well-formed. It's probably more accurate to say that something about how it enforces a specific data structure.

```suggestion
Your data model ensures that your data complies with a specific structure, contains
```